crates/turbopack-dev-server/src/update/stream.rs

```diff
@@ -55,7 +56,7 @@ async fn get_update_stream_item(
             plain_issues.push(
                 FatalStreamIssue {
                     resource: resource.to_string(),
-                    description: Vc::cell(format!("{}", PrettyPrintError(&e))),
+                    description: StyledString::String(format!("{}", PrettyPrintError(&e))).cell(),
```

Comment:
We might want a helper to pretty print an error directly into a styled string, so we can use formatting during pretty printing. But that's something for a follow-up PR